### ISSUE TITLE
please update for openssh 6.7?

### ISSUE THREAD

**Issue by yarikoptic** [rapier1/hpn-ssh-archive#1]
OpenSSH 6.7 released October 6, 2014

**Comment by rapier1** [comment]
We will be updating as soon as possible. Unfortunately other project 
have completely absorbed all available cycles. I expect to be able to 
get to this in the second week of January. I'm really sorry for the delay.

**Comment by FutureCow** [comment]
Do you have any status update about this?

**Comment by vapier** [comment]
work has moved to https://github.com/rapier1/openssh-portable ... 6.8p1 can be found there

**Comment by christophprokop** [comment]
I'm beginning to get confused which website actually holds the "official" hpn-patches:

[1] http://www.psc.edu/index.php/hpn-ssh  (patches up to 6.3p1)
[2] http://sourceforge.net/projects/hpnssh/files/?source=navbar  (patches up to 6.6p1)
[3] https://github.com/rapier1/openssh-portable/releases (patch 6_8_P1)

Would you be so kind to provide some kind of link or similar on site [1] to verify the integrity of repository [3] ?

Also, some downloads provide patch files onto the source tarballs from openssh.org.
linked within [1]: http://www.psc.edu/index.php/hpn-ssh-patches/hpn-14-kitchen-sink-patches/viewcategory/24
while this more recently released download page
linked under "releases" within [3] https://github.com/rapier1/openssh-portable/releases
seems to provide entire modified source trees.

Thirdly, is there a pgp signature/key (or the like) to verifiy the integrity of the downloads?

Sorry if i missed something obvious. Thanks for any insights.

**Comment by vapier** [comment]
the repo i referred to holds the latest official work.  Chris was going to update the other pages to refer to the github repo.

**Comment by christophprokop** [comment]
Thanks for your fast response! And thank you even more for putting effort into the hpn-feature in general!
Will you also provide a patch file which applies to these openssh source-tarballs?
http://www.openssh.com/portable.html#ftp

**Comment by rapier1** [comment]
Yes indeed. The github repo is going to be the canonical source but 
patches will also be made available on sourceforge. That just seems to 
be a good location for a lot of end users.

Chris

**Comment by christophprokop** [comment]
Alright, now that is good news! Thanks for clarifying!
It would be really helpful to link to the github repo from http://www.psc.edu/index.php/hpn-ssh
Otherwise it could be anyone setting up an account called "rapier1" and claim that repo to be "official" ;)

**Comment by bdrewery** [comment]
Can you please enable issues in the new repo?

**Comment by christophprokop** [comment]
Hi there!
Any news about: https://github.com/rapier1/hpn-ssh/issues/1#issuecomment-85000720 ?

**Comment by bdrewery** [comment]
FWIW for 6.8 I have ported the NONECIPHER and DYNAMICWINDOWS features in https://github.com/freebsd/freebsd-ports/blob/master/security/openssh-portable/files/extra-patch-hpn. There is nothing FreeBSD-specific in there. My modifications were adding build-time configure options and wrapping the features in `#ifdef` blocks and fixing what I submitted in https://github.com/rapier1/openssh-portable/pull/2. You'll need to build with `--with-hpn` and `--with-nonecipher` to enable each feature.

**Comment by jeeger** [comment]
Sorry for digging this up again, but are there HPN patches available for 6.7p1 anywhere? I've searched around, but only found 6.8 and newer, and 6.6 and older. No 6.7 HPN patches found. Am I missing something?

**Comment by vapier** [comment]
there is no official release for 6.7p1.  Gentoo's unofficial patchset is here: https://dev.gentoo.org/~vapier/dist//openssh-6.7p1-hpnssh14v5.tar.xz

then again, you really shouldn't be using old versions with known security problems.
